ios/Classes/SharedStorage.swift

```diff
@@ -18,38 +19,84 @@ public enum SharedStorage: Int {
 }
 
 /// Move file at [filePath] to shared storage [destination] with optional [directory]
-public func moveToSharedStorage(filePath: String, destination: SharedStorage, directory: String) -> String? {
```

Comment:
Shouldn't you check for the existence of the file in add circumstances, i.e. whether you use your images/photos approach or the old 'fake' one? I think it's best to leave that check here, and return if the file doesn't exist